wordpress-mobile/WordPress-iOS
# Jetpack Section: Threat Details screen

Part of #15190 
WPKit: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/337

### Description:
- Added Threat Details screen
- TODO: hook up buttons
- TODO: display threat context

### To test:
1. Launch the app, tap on the My Site, tap on a site with Jetpack Scan and also has Threats
2. Tap on a Threat cell
    - ✅ Should navigate to a new Threat Details screen
    - ✅ Threat Details screen should be scrollable (if the content height is bigger than the screen height)
3. Launch Calypso and go to Jetpack > Scan, and compare info shown in Threat Details screen to Calypso
    - ✅ Info displayed on the Threat Details screen should match info displayed on Calypso
    
### Example screenshots:
![Simulator Screen Shot - iPhone 12 Pro - 2021-01-27 at 15 34 04](https://user-images.githubusercontent.com/6711616/105952603-24f03080-60b5-11eb-8506-4c8b6fe9d8c4.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-01-27 at 15 34 09](https://user-images.githubusercontent.com/6711616/105952610-2588c700-60b5-11eb-857e-3a4f2c8f9476.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-01-27 at 15 33 59](https://user-images.githubusercontent.com/6711616/105952599-23bf0380-60b5-11eb-904c-3be86899097c.png)
-- | -- | --

PR submission checklist:
